StudyPactOnboardingViewController.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  StudyPactOnboardingViewController.swift
+//  berkeley-mobile
+//
+//  Created by Eashan Mathur on 10/17/20.
+//  Copyright © 2020 ASUC OCTO. All rights reserved.
+//
+
+import UIKit
+
+
+class StudyPactOnboardingViewController: UIViewController {
+
+    let pageNumber = 0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        let signUpLabel = UILabel()
+        let bearImage = UIImageView()
+        let numberImage = UIImageView()
+        let descriptionText = UILabel()
```

Comment:
try to initialize these using a
`let signUpLabel: UILabel = { configure and return the label }()`
style for consistency.
check out OverviewCardView lines 211-366 for reference